docs/release-notes/release-notes-0.13.1.md

```diff
@@ -6,6 +6,18 @@ Scripts received as part of an upfront shutdown script [are now properly
 sanitized](https://github.com/lightningnetwork/lnd/pull/5369) to ensure
 widespread relay of potential cooperative channel closures.
 
+## Wallet Unlocking
+
+[A new flag has been added](https://github.com/lightningnetwork/lnd/pull/5457)
```

Comment:
Oh, I forgot about the release notes. Should we perhaps add a check to the GitHub actions that the release notes were updated?

Reply:
Yeh it's been on my TOOD list for a while, but I haven't got around to finalizing the hacky bash script I started on lol. Thinking minimally it should: fail the build if based on the milestone, and the release notes file for that milestone, the PR number isn't in the raw markdown. So minimally you just need to mention the PR itself, but ideally the contributor adds more detailed information about the change.